src/common/enums/ui/button-color.enum.ts

```diff
@@ -0,0 +1,5 @@
+enum ButtonColor {
```

Comment:
We should not have a color and a type params separately, it should go as one, that defines the style for our custom button. Also it should be placed within the button component folder at least